datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -158,6 +158,34 @@ def default_json_event_encoding(o):
     raise TypeError
 
 
+class DbRow:
```

Comment:
Do we really need this wrapper class? 

Seems we could keep rows as plain dictionaries and simply add these new pieces of metadata as new "columns" in the row without having to create a new wrapper type?

Reply:
Yeah, that's definitely a valid approach and would simplify some logic in the integrations. I figured having a wrapper interface would make for a nicer experience as opposed to something like returning tuples everywhere. However, injecting the metadata as "columns" isn't something I thought about initially, thanks!